src/WireMock.Net/Matchers/JSONPathMatcher.cs

```diff
@@ -123,7 +123,12 @@ private double IsMatch(JToken jToken)
     {
         var array = ConvertJTokenToJArrayIfNeeded(jToken);
 
-        return MatchScores.ToScore(_patterns.Select(pattern => array.SelectToken(pattern.GetPattern())?.Any() == true).ToArray(), MatchOperator);
+        // The SelectToken method can accept string path to a child token ( i.e. “Manufacturers[0].Products[0].Price”) in that cat it will return a JValue(some type) which not implement the IEnumerable interface.

```

Comment:
Change the comment to this?
```
// The SelectToken method can accept a string path to a child token ( i.e. "Manufacturers[0].Products[0].Price"). In that case it will return a JValue (some type) which does not implement the IEnumerable interface.
```